package.json

```diff
@@ -17,6 +17,7 @@
     "@aws-sdk/client-s3": "^3.241.0",
     "@aws-sdk/client-ses": "^3.245.0",
     "@aws-sdk/util-dynamodb": "^3.241.0",
+    "@babel/helper-remap-async-to-generator": "^7.22.20",
```

Comment:
nit: Not sure if this is used anywhere? Should remove from dependencies if not used